.gitignore

```diff
@@ -41,3 +41,10 @@ TagManager.log.lck
 TagManager.log.lck.1
 ViewNoteCommand.log.lck
 data/
+TagManager.log.2
+TagManager.log.2.lck
+TagManager.log.3
+TagManager.log.3.lck
+TagManager.log.4
+TagManager.log.5
+TagManager.log.5.lck
```

Comment:
There is a way to gitignore all the TagManager.log files
https://www.atlassian.com/git/tutorials/saving-changes/gitignore